frontend/src/lib/components/LemonButton/index.tsx

```diff
@@ -9,6 +9,8 @@ import './LemonButton.scss'
 export type LemonButtonPopup = Omit<PopupProps, 'children'>
 export interface LemonButtonPropsBase extends Omit<LemonRowPropsBase<'button'>, 'tag' | 'type' | 'ref'> {
     type?: 'default' | 'primary' | 'secondary' | 'stealth' | 'highlighted'
+    /** Whether hover style should be applied, signaling that the button is held active in some way. */
+    held?: boolean
```

Comment:
Hmm, yeah, `held` is pretty ambiguous. `applyHoverStyle` seems a tad too imperative though, React is more about being declarative. Renamed this to `active` instead. It may be a slight dissonance between the CSS `:active` and this, but really the prop seems pretty clear this way.